airbyte-integrations/connectors/source-twilio/unit_tests/test_streams.py

```diff
@@ -1,327 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-from contextlib import nullcontext
-from unittest.mock import patch
-
-import pendulum
-import pytest
-import requests
-from airbyte_cdk.sources.streams.http import HttpStream
-from freezegun import freeze_time
-from source_twilio.auth import HttpBasicAuthenticator
-from source_twilio.source import SourceTwilio
-from source_twilio.streams import (
-    Accounts,
-    Addresses,
-    Alerts,
-    Calls,
-    DependentPhoneNumbers,
-    MessageMedia,
-    Messages,
-    Recordings,
-    TwilioNestedStream,
-    UsageRecords,
-    UsageTriggers,
-)
-
-TEST_CONFIG = {
-    "account_sid": "airbyte.io",
-    "auth_token": "secret",
-    "start_date": "2022-01-01T00:00:00Z",
-    "lookback_window": 0,
-}
```

Comment:
We must keep the same behavior to the new connector and not delete all unit tests.

Reply:
gotcha